test/blocks/section-metadata/section-meta.test.js

```diff
@@ -20,6 +20,15 @@ describe('Section Metdata', () => {
     expect(sec.classList.contains('has-background')).to.be.true;
   });
 
+  it('Handles background image focal point', () => {
+    it('gets section metadata', () => {
```

Comment:
I don't think you need two `it` blocks here.

Reply:
done